tutorials/nginx-reverse-proxy-docker.md

```diff
@@ -0,0 +1,392 @@
+---
+title: Running an NGINX Reverse Proxy with Docker and Let's Encrypt on Google Compute Engine
+description: Learn to run multiple services on a Google Compute Engine virtual machine using Docker and a reverse proxy.
```

Comment:
virtual machine -> instance

Can we instead say:

> Learn to serve multiple websites simultaneously in a single Compute Engine instance with Docker and NGINX

? Or is running multiple websites with different domains on port 80 simultaneously not a goal?